peer-review-bundle-spec.md

```diff
@@ -0,0 +1,147 @@
+# Qwiklabs PeerReview Bundle Specification
```

Comment:
Is this page specific for only peer review? Or would it also cover ML-assisted/based reviews?

Reply:
yes, it is only peer review, and it opens to evolving as we reach different milestone.

right now, ML review is still in experimental stage, and we are still exploring different technical challenges:
https://docs.google.com/document/d/13DGgidTJEqOedq0YaR4PH0B-1mrYhHbRqLd6hubOCqM/edit?resourcekey=0-QCzge5PnQ4MDOCKQXnMp5g#heading=h.hir0blkcmi8v
which might take unknow years (or 2 years) to mature.

Reply:
I am asking because the `entity_type: PeerReview`. Should we consider making this something a bit more generic that covers the case of ML-assisted model, e.g., `ReviewableActivity` or `GradableActivity` etc?

Reply:
hmm..we had a brainstorm session about new learning entity name
https://docs.google.com/presentation/d/1H6W3OxoQBpnnEQYeyuDgoJb9uH5B0j4E7bOIJ_BObm8/edit#slide=id.g177b04927d1_0_1
p5, p6 lists the candidate names, and why we make the decision about "Peer Review"

let me know if you like have another follow up session about this. thank you.

Reply:
update them to PeerAssignment with the vote from Ant, Dave, and Carine